src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -2601,6 +2607,8 @@ public partial class Exception : System.Runtime.Serialization.ISerializable
         public virtual string StackTrace { get { return default(string); } }
         public System.Reflection.MethodBase TargetSite { get { return default(System.Reflection.MethodBase); } }
         protected event System.EventHandler<System.Runtime.Serialization.SafeSerializationEventArgs> SerializeObjectState { add { } remove { } }
+        [System.Runtime.CompilerServices.FriendAccessAllowedAttribute]
+        internal void AddExceptionDataForRestrictedErrorInfo(string restrictedError, string restrictedErrorReference, string restrictedCapabilitySid, object restrictedErrorObject, bool hasrestrictedLanguageErrorObject = false) { }
```

Comment:
> If we wanted a library to work for UWP F5 on CoreCLR that we shipped at RTM last summer it cannot have a reference to S.P.CoreLib

I do not think we want this. It just makes our test matrix to explode and I do not think it matches, and it provides very little value. What's wrong with saying that if you want to use latest System.Runtime.WindowsRuntime, you have to use latest CoreCLR as well?